languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -14776,6 +14776,21 @@ USA
     <!-- Grammar -->
     <!-- ====================================================================== -->
     <category id="GRAMMAR" name="Grammar" type="grammar">
+        <rule id="SEEMING_SEEMS" name="confusion of is seeming/seems" default="temp_off">
+            <antipattern>
+                <token>am</token>
+                <token>seeming</token>
+                <token>like</token>
+            </antipattern>
+            <pattern>
+                <marker>
+                    <token inflected="yes">be</token>
+                    <token>seeming</token>
+                </marker>
+            </pattern>
```

Comment:
Could there be FPs for "seeming" used as an adjective?

_There are seeming differences between these instances, but on closer inspection, they turn out to be negligible."